cert/openssl.cnf

```diff
@@ -509,6 +509,30 @@ crlDistributionPoints=@crl_section
 #FCCID
 #SERIAL
 
+[ cbsd_req_device_a_sign ]
+
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid:always,issuer
+basicConstraints = CA:FALSE
+keyUsage = critical, digitalSignature, keyEncipherment, cRLSign
+extendedKeyUsage = clientAuth
+certificatePolicies=@cps,ROLE_CBSD
+crlDistributionPoints=@crl_section
+FCCID=ASN1:UTF8String:test_fcc_id_a
+SERIAL=ASN1:UTF8String:test_serial_number_a
+
+[ cbsd_req_device_c_sign ]
+
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid:always,issuer
+basicConstraints = CA:FALSE
+keyUsage = critical, digitalSignature, keyEncipherment, cRLSign
+extendedKeyUsage = clientAuth
+certificatePolicies=@cps,ROLE_CBSD
+crlDistributionPoints=@crl_section
+FCCID=ASN1:UTF8String:test_fcc_id_c
+SERIAL=ASN1:UTF8String:test_serial_number_c
+
```

Comment:
I don't think you need to add these two req_sign (cbsd_req_device_a_sign and cbsd_req_device_c_sign), FCCID and SERIAL are optionals
Why you don't use cbsd_req_sign?